### PR TITLE
Fixed a bug in paperSize.go toMillimetres, which was returning d.Heig…

### PR DESCRIPTION
**Pull request by cewitte**
…ht twice instead of d.Width and d.Height; and fixed ToMillimeters and ToCentimeters spelling errors.

### CODE REVIEW

File: pkg/pdfcpu/paperSize.go [hunk 1]
```diff
@@ -49,14 +49,14 @@ func (d Dim) ToInches() Dim {
 	return Dim{d.Width * userSpaceToInch, d.Height * userSpaceToInch}
 }
 
-// ToCentimetres converts d to centimetres.
-func (d Dim) ToCentimetres() Dim {
+// ToCentimeters converts d to centimetres.
```
Comment (hhrutter) [inline pkg/pdfcpu/paperSize.go]: I am ok with using the American spelling across the board for centimeters and millimeters, but this has to be done consistently for all occurences in the code base including all comments.

For function name changes please also fix all references eg. like in context.go

Comment (cewitte) [reply pkg/pdfcpu/paperSize.go]: Thanks for your comments.

I was looking into it and there are a lot of cases like that, from switch/case scenarios to func calls in many places. I could make the changes, but there's a huge risk of breaking too much of other people's code. From godoc.org it felt like an isolated thing, but in fact it's not (sorry for the lack of proper investigation).

Please feel free to reject the changes - the small bug fix is still valid though - or let me know if you still wanna change all occurrences.

Comment (hhrutter) [reply pkg/pdfcpu/paperSize.go]: I am happy to merge this in if you limit the change to the obvious bugfix.
I am going to take care of the spelling issue later on myself.

**Comment by hhrutter** [comment]
Do you think you can squash your contribution into one [signed commit](https://help.github.com/en/github/authenticating-to-github/signing-commits)?

**Pull request merged**